h5netcdf/utils.py

```diff
@@ -29,6 +33,38 @@ def __repr__(self):
         return f"{type(self).__name__}({self._mapping!r})"
 
 
+def write_classic_string_dataset(gid, name, value, shape, chunks):
```

Comment:
This function should be aligned with the other util functions recently introduced. Make it private and strip off anything unrelated.